looker/definitions/firefox_desktop.toml

```diff
@@ -9,6 +11,41 @@ numerator = "socket_crash_count_v1.sum"
 denominator = "socket_crash_active_hours_v1.sum"
 
 
+## OKRs example
+
+[metrics.desktop_engagement_dau_v1]
+select_expression = "SUM(dau)"
+data_source = "desktop_engagement_v1"
+friendly_name = "(non-official) DAU"
+description = "(non-official) Daily active users on desktop based on engagment data"
+
+[metrics.desktop_engagement_wau_v1]
+select_expression = "SUM(wau)"
+data_source = "desktop_engagement_v1"
+friendly_name = "(non-official) WAU"
+description = "(non-official) Weekly active users on desktop based on engagment data"
+
+[metrics.desktop_engagement_mau_v1]
+select_expression = "SUM(mau)"
+data_source = "desktop_engagement_v1"
+friendly_name = "(non-official) MAU"
+description = "(non-official) Monthly active users on desktop based on engagment data"
+
+
+[metrics.desktop_engagement_dau_v1.statistics.sum]
+[metrics.desktop_engagement_wau_v1.statistics.sum]
+[metrics.desktop_engagement_mau_v1.statistics.sum]
+
+[metrics.desktop_engagement_dau_v1.statistics.ratio]
+numerator = "desktop_engagement_dau_v1.sum"
+denominator = "desktop_engagement_mau_v1.sum"
+
+[metrics.desktop_engagement_dau_v1.statistics.rolling_average]
+window_sizes = [1, 7, 28]
```

Comment:
adding these metric definitions here as they are probably not considered official yet (?). We can move them to the `definitions/` folder later